### PR TITLE
Restrict share option only to staff

### PR DESCRIPTION
**Pull request by prabhanshu**
##### Acceptance Criteria

1. An author can choose to make the share button available to 'staff only' to make the video anonymous, but will no longer have the option to set the "Share the video' with 'All' option.
2. The dropdown option called 'staff only' is renamed to 'On'. 
3. The 'Share the video' dropdown options will be : 'Off (Default)' and 'On'
4. The share button continues to provides an embeddable URL and code so staff/authors can embed the experience into a site that supports iFrames.
5. When clicking the Share button from the video player, this section is completely remove" The player is governed by this license. Please read it. By copying or displaying this player, you agree to abide by this license. If you do not agree, do not copy or display the player.
6. Definition of Done is met.

#### Definition of done:
- [ ] Title of the pull request is clear and informative
- [ ] Add pull request hyperlink to relevant TFS items
- [ ] For large or complex change: schedule an in-person review session
- [ ] This change has appropriate test coverage
- [ ] Get at least two approvals

#### Reminders DURING merge
1. If you're merging from a short-term (feature) branch into a long-term branch (like dev, release, or master) then "Squash and merge" to keep our history clean.
1. If merging from two longterm branches (like cherry picks from upstream, dev to release, etc) then "Create merge commit" to preserve individual commits.

[//]: # ( fyi: This content was heavily inspired by )
[//]: # ( 1 Our team's policies and processes )
[//]: # ( 2 https://github.com/sprintly/sprint.ly-culture/blob/master/pr-template.md )
[//]: # ( 3 The book "The Checklist Manifesto: How to Get Things Right" by Atul Gawande )
[//]: # ( 4 https://github.com/Azure/azure-event-hubs/blob/master/.github/PULL_REQUEST_TEMPLATE.md )

### CODE REVIEW

**Comment by manikarthikk** [comment]
can you update the title

**Comment by prabhanshu** [comment]
@manikarthikk done

File: azure_media_services/templates/player.html [hunk 1]
```diff
@@ -49,10 +49,6 @@
                 </select>
               </div>
             </div>
-            <div class="embed-tou">
```
Comment (vivalams) [inline azure_media_services/templates/player.html:52]: Its not very clear from the PR description and the acceptance criteria if this text that is removed has to be replaced with new text?

Comment (prabhanshu) [reply azure_media_services/templates/player.html:52]: Acceptance Criteria mentions that:
"When clicking the Share button from the video player, this section is completely remove" The player is governed by this license. Please read it. By copying or displaying this player, you agree to abide by this license. If you do not agree, do not copy or display the player."

**Review by manikarthikk — APPROVED** [review]

**Review by vivalams — APPROVED** [review]

**Pull request merged**